tools/tools.yml

```diff
@@ -302,6 +302,11 @@
       badges: pypi, conda
       builtons: [leaflet, plotly]
 
+    - repo: JetBrains/lets-plot
+      sponsors: [jetbrains]
+      site: https://lets-plot.org
```

Comment:
Thanks! Please update the existing entry for lets-plot to match this one. My guess is that the `site` listed in this entry is better than in the other one.

Reply:
Hi! It's waiting :) see https://github.com/pyviz/pyviz.org/pull/160

Reply:
Perfect, thanks!